server/messaging/reporting.py

```
@@ -18,12 +18,12 @@
 INSTALL_PATH = os.getenv("NETALERTX_APP", "/app")
 sys.path.extend([f"{INSTALL_PATH}/server"])
 
-from helper import (
+from helper import (  # noqa: E402 [flake8 lint suppression]
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)